### DISCUSSION TITLE
Showing Hour of the day using a Stat Card

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#1964] [Q&A] [answered]
Hi!I would like to show a quantity which represents an hour of the day. I tried using a Wide Gauge Stat Card as I want to display a value as well as a clock or similar graphic. 

![image](https://user-images.githubusercontent.com/117831190/236116361-dc8cc434-56ea-4402-b0db-fb979edec62d.png)

Is it possible to replace the progress gauge with another graphic that represents a clock?Or even if I just display the gauge without the progress % value or change the range from 0 to 1 to 0 to 24, It will do the job! The Graphics - Clock example seemed a bit complex to implement for this simple task. Thank you!

**Comment by mturoci** [comment, marked as answer]
> Is it possible to replace the progress gauge with another graphic that represents a clock?

Not possible. You will need to create your own card for that using ui.inline with text on RHS and clock (for example SVG) on the LHS. It's not an easy task though and requires a lot of SVG knowledge just so you know.

**Reply by n-srinidhi** [reply]
Yes!Is there any other component you can suggest to best display this information?Something that can display hour of the day along with an icon or graphic? Because using a plain stat_card looks very flat.

**Reply by n-srinidhi** [reply]
@mturoci I used stats component within a form card instead. Thank you!

**Reply by mturoci** [reply]
You are faster than me :) Stats is exactly what you are looking for if you need a text + icon combo.
